### DISCUSSION TITLE
How to run marker.kt

### DISCUSSION THREAD

**Discussion by xichquylk** [patrykandpatrick/vico#678] [Questions] [answered]
Hi,
I have a problem with marker.kt file in chart, when I create a Marker.kt file as below image.
I use version 1.14.0
Can you help me how to use this?
![image](https://github.com/patrykandpatrick/vico/assets/32903810/16c67b45-dbe4-49c7-8de6-686035758ee1)

**Comment by patrickmichalik** [comment, marked as answer]
Hello! This `Marker.kt` revision is from the sample app for Vico 2. Please switch to the tag corresponding to your Vico version. The tag for Vico 1.14.0 is `v1.14.0`.

**Reply by xichquylk** [reply]
Thanks so much!

**Reply by patrickmichalik** [reply]
You’re welcome!
